packages/suite/src/views/wallet/transactions/TransactionList/TransactionList.tsx

```diff
@@ -68,36 +74,37 @@ export const TransactionList = ({
     );
 
     useEffect(() => {
-        if (anchor && !hasFetchedAll) {
-            dispatch(
-                fetchAllTransactionsForAccountThunk({
-                    accountKey: account.key,
-                    noLoading: true,
-                }),
-            );
-            setHasFetchedAll(true);
+        if (anchor && !fetchedAll) {
+            fetchAll();
```

Comment:
we pay for each request on cardano/solana/ripple. Maybe we can limit anchors (tx notification) for Blockbook only

Reply:
Good idea.

Reply:
Created issue: https://github.com/trezor/trezor-suite/issues/17024